source/manual/configure-github-repo.html.md

```diff
@@ -11,7 +11,7 @@ Repositories in GOV.UK must:
 - Have the [GOV.UK CI Bots][govuk-ci-bots-team] and [GOV.UK Production][govuk-production-team] teams as `Admin`
 - Have a good description
 - Link to relevant documentation
-- Be tagged with [`govuk`](https://github.com/search?q=topic:govuk)
+- Be tagged with [`govuk`][govuk-topic]
```

Comment:
GitHub has the concept of "tags" so we could make it clearer that this is a "topic" instead:
```suggestion
- Have the [`govuk`][govuk-topic] topic
```
or
```suggestion
- Be tagged with the [`govuk`][govuk-topic] topic
```

Reply:
Adding a link to the docs on how to do it would also be good: https://docs.github.com/en/github/administering-a-repository/classifying-your-repository-with-topics